pkg/datapath/iptables/iptables.go

```diff
@@ -399,6 +399,8 @@ func (m *IptablesManager) Init() {
 		if option.Config.EnableIPv6 {
 			log.WithError(err).Warning(
 				"IPv6 is enabled and ip6tables modules could not be initialized")
+			log.WithError(err).Warning(
+				"Try loading ip6_tables, ip6table_mangle, ip6table_raw and ip6table_filter modules.")
```

Comment:
nit, its a bit odd to have back to back log messages like this IMO. Just put this on the same line as above.

Reply:
Sure. I was originally thinking to do that only.